src/tests/unit/tests/injected/analyzers/tab-stops-handler.test.ts

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { TabStopsHandler } from 'injected/analyzers/tab-stops-handler';
+import { AutomatedTabStopRequirementResult } from 'injected/tab-stop-requirement-result';
+import { TabStopsRequirementEvaluator } from 'injected/tab-stops-requirement-evaluator';
+import { TabbableElementGetter } from 'injected/tabbable-element-getter';
+import { IMock, Mock } from 'typemoq';
+
+describe(TabStopsHandler, () => {
+    let tabbableElementGetterMock: IMock<TabbableElementGetter>;
+    let requirementEvaluatorMock: IMock<TabStopsRequirementEvaluator>;
+
+    const firstTabStop = { innerText: 'first tab stop' } as HTMLElement;
+    const currentTabStop = { innerText: 'new tab stop' } as HTMLElement;
+    const anotherTabStop = { innerText: 'another tab stop' } as HTMLElement;
+    let tabbableElementsStub: HTMLElement[];
+    const resultStub: AutomatedTabStopRequirementResult = {
+        selector: [' selector'],
+    } as AutomatedTabStopRequirementResult;
+
+    let testSubject: TabStopsHandler;
+
+    beforeEach(() => {
+        tabbableElementsStub = [
+            { innerText: 'tabbabble element 1' },
+            { innerText: 'tabbabble element 2' },
+        ] as HTMLElement[];
+        tabbableElementGetterMock = Mock.ofType<TabbableElementGetter>();
+        requirementEvaluatorMock = Mock.ofType<TabStopsRequirementEvaluator>();
+
+        tabbableElementGetterMock
+            .setup(t => t.getRawElements())
+            .returns(() => tabbableElementsStub);
+
+        testSubject = new TabStopsHandler(
+            requirementEvaluatorMock.object,
+            tabbableElementGetterMock.object,
+        );
+        testSubject.initialize();
+    });
+
+    afterEach(() => {
+        requirementEvaluatorMock.verifyAll();
+        tabbableElementGetterMock.verifyAll();
+    });
+
+    it('getTabbabbleFocusOrderResults() calls requirement evaluator with tabbabble elements', () => {
```

Comment:
It looks like we don't have any tests for `getKeyboardNavigationResults`. It's exercised as a side effect of other tests but personally I'd like to have a standalone test for it as well.